Sources/FluentKit/Properties/Siblings.swift

```diff
@@ -34,8 +34,45 @@ public final class Siblings<From, To, Through>: AnyProperty
         return self
     }
 
+    // MARK: Checking state
+
+    public func isAttached(to: To, on database: Database) -> EventLoopFuture<Bool> {
+        guard let toID = to.id else {
+            fatalError("Cannot attach unsaved model.")
+        }
+
+        return isAttached(toID: toID, on: database)
```

Comment:
`self.` everywhere, please

Reply:
Not a fan of `self.` where it's not really needed, but not my rules 🤷‍♂